background/lib/posthog.ts

```diff
@@ -26,7 +26,7 @@ export const isOneTimeAnalyticsEvent = (
   return Object.values<string>(OneTimeAnalyticsEvent).includes(eventName)
 }
 
-const POSTHOG_PROJECT_ID = "11112"
+const POSTHOG_PROJECT_ID = "0"
```

Comment:
what is this ID?

Reply:
It's a default ID, used in the posthog endpoint URL. Probably won't work, but Posthog isn't accessed with this commit anyways.

Reply:
Does it need to change as part of this PR? I'd prefer to only change what needs to change

Reply:
This PR also disables analytics. I updated the description to reflect that.